synapse/handlers/account_data.py

```diff
@@ -174,6 +174,7 @@ def get_current_key(self, direction: str = "f") -> int:
     async def get_new_events(
         self,
         user: UserID,
+        *,
```

Comment:
Why make these keyword-only args?

Reply:
To start with, I was trying to figure out why the signatures of the methods of the subclass were not compatible with the signature of the parent's method, then left them in because it seemed to match up with how these functions are called...

It looks like the `*` can be removed and no further complaints, so I'm happy to take them out.